sonar-dotnet-shared-library/pom.xml

```diff
@@ -63,9 +63,9 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.13.1</version>
+      <groupId>org.junit.jupiter</groupId>
```

Comment:
I initially updated it for parameterized tests (junit 4 offers a pretty bad experience), but then realized it's better to not do parameterized tests for my case (too complex)

however, it's not hurting to be on the latest

Reply:
0% coverage may be due to this change, I will revert the `pom.xml` changes